### ISSUE TITLE
Documentation needed

### ISSUE THREAD

**Issue by vladzur** [spiritdead/resque#1]
This is a very interesting refactor of php-resque but i miss a little  documentation on how to use it and testing it. ¿Can you provide one?

**Comment by spiritdead** [comment]
@vladzur  yes, is something i have to finish and the codeception too, i will try find time to do this

**Comment by roynasser** [comment]
I agree, how similar in usage is it to chris's version? is this drop-in for jobs workers and job queuing?

**Comment by spiritdead** [comment]
is similar but im moving this to OOP and to work with yii2 framework
